### ISSUE TITLE
apagar um clone de um projeto na maquina?

### ISSUE THREAD

**Issue by fernandofabril** [gustavoguanabara/git-github#120]
Professor para eu apagar um clone de um projeto é só apagar a pasta na maquina?

**Comment by webkaiyo** [comment]
Você deve deletar a pasta **junto** com a pasta `.git` (que está oculta dentro do seu repositório). Se você não deletar esta pasta, o repositório continuará a existir (localmente, é claro).

**Comment by leocunhabr** [comment]
Sim, basicamente, mais fale mais sobre isso, a pergunta está muito reduzida, não tenho como saber se é só isso mesmo.
